Vue Component.sublime-syntax.yaml-macros

```diff
@@ -0,0 +1,428 @@
+    - include: langs
+    - include: template-tag
+
+  tag-attributes: !prepend
+    - include: vue-directive
+
+  template-tag:
+    - match: '(?i)(<)(template)\b'
+      captures:
+        0: meta.tag.template.begin.html
+        1: punctuation.definition.tag.begin.html
+        2: entity.name.tag.template.html
+      push:
+        - match: (?i)(</)(template)(>)
+          captures:
+            0: meta.tag.template.end.html
+            1: punctuation.definition.tag.begin.html
+            2: entity.name.tag.template.html
+            3: punctuation.definition.tag.end.html
+          pop: true
+        - match: '>'
+          scope: meta.tag.template.begin.html punctuation.definition.tag.end.html
+          push: mustache-template
+        - match: ''
+          push:
+            - meta_scope: meta.tag.template.begin.html
+            - match: '(?=>)'
+              pop: true
+            - include: tag-attributes
+
+  mustache-template:
+    - match: (?=</template)
+      pop: true
+    - match: '{{'
+      scope: punctuation.definition.template.begin.html
+      embed: scope:source.js
+      escape: '}}'
+      escape_captures:
+        0: punctuation.definition.template.end.html
+    - include: main
+
+  vue-directive:
+    - match: \b(v-[\w-]+)\b
+      scope: entity.other.attribute-name.html
+      push: js-attribute-value
+    - match: \B(:|@)([\w-]+)\b
+      captures:
+        1: punctuation.definition.attribute.html
+        2: entity.other.attribute-name.html
+      push: js-attribute-value
+
+  js-attribute-value:
+    - match: '='
+      scope: punctuation.separator.key-value.html
+      set: js-string
+    - include: else-pop
+
+  js-string:
+    - match: '"'
+      scope: punctuation.definition.string.begin.html
+      embed: scope:source.js
+      escape: '"'
+      escape_captures:
+        0: punctuation.definition.string.end.html
+    - match: "'"
+      scope: punctuation.definition.string.begin.html
+      embed: scope:source.js
+      escape: "'"
+      escape_captures:
+        0: punctuation.definition.string.end.html
+    - include: else-pop
+
+  langs:
+    - match: (?:^\s+)?(<)((?:script))\b(?=[^>]*lang\s*=\s*(['"])coffee\1?)
+      captures:
+        0: meta.tag.script.begin.html
+        1: punctuation.definition.tag.begin.html
+        2: entity.name.tag.script.html
+      push:
+        - match: (?i)(-->)?\s*(</)(script)(>)
+          captures:
+            0: meta.tag.script.end.html
+            1: comment.block.html punctuation.definition.comment.html
+            2: punctuation.definition.tag.begin.html
+            3: entity.name.tag.script.html
+            4: punctuation.definition.tag.end.html
+          pop: true
+        - match: '(>)\s*(<!--)?'
+          captures:
+            1: meta.tag.script.begin.html punctuation.definition.tag.end.html
+            2: comment.block.html punctuation.definition.comment.html
+          embed: scope:source.coffee
+          escape: (?i)(?=(-->)?\s*</script)
```

Comment:
The same comment applies to the `-->` in the lookahead here.

Reply:
Fixed. Thank you!